src/registerToolbar.ts

```diff
@@ -86,9 +86,10 @@ export default function registerToolbar() {
     ToolbarItemFactory.register({
         name: ToolbarItemName.ClearRegions,
         tooltip: "Clear Regions",
-        icon: "fa-trash-alt",
+        icon: "fa-ban",
         group: ToolbarItemGroup.Regions,
         type: ToolbarItemType.Action,
+        accelerators: ["Ctrl+Delete"],
```

Comment:
I thought this only remove selected regions, or does it remove all?

Reply:
Delete would remove selected regions, ctrl delete removes all